### DISCUSSION TITLE
Site looks slightly different on local versus github pages

### DISCUSSION THREAD

**Discussion by elizzakai** [community/community#36824] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, I noticed  that my website on local, in terms of the CSS is slightly different. It isnt super obvious, but for example, the buttons do not have their hovering effect anymore.

I inspected the site on web and saw some 404 errors for my variables and global.css file I think, but there isnt anything specific with that. It just notes the first line?

But yea, feel free to help me out if you have the time. I just want to get this fixed before adding to my site.

**Link to repo: https://github.com/elizzakai/my_Portfolio** 
**Github Pages site: https://elizzakai.github.io/my_Portfolio/**

**Comment by airtower-luna** [comment, marked as answer]
There are two parts to the problem:

* You're building the site using the default Jekyll build, but your repository contains a plain HTML site, not a Jekyll site. Usually that'd work, but:
* Jekyll ignores files starting with an underscore, like your `_globals.css` and `_variables.css`.

The easiest two solutions are (either should work):

* Rename those CSS files so their names don't start with an underscore.
* Add a `.nojekyll` file to the root of your repository to disable the Jekyll build and deploy your page as-is.

**Reply by elizzakai** [reply]
Ah okay, just fixed it thanks!

**Reply by airtower-luna** [reply]
Glad I could help! Marking the answer would be appreciated. :slightly_smiling_face:

**Reply by RonenTsirlin** [reply]
Thanks a lot! fixed my problem too!
